FluentTerminal.Models/Extensions.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentTerminal.Models.Enums;
+using Windows.ApplicationModel.Resources;
+
+namespace FluentTerminal.Models
+{
+    public static class Extensions
+    {
+        public static string GetResource(this string resource) =>
+            ResourceLoader.GetForCurrentView().GetString(resource.Replace('.', '/'));
+
+        public static SshConnectionInfoValidationResult GetValidationResult(this ISshConnectionInfo sshConnectionInfo,

```

Comment:
not a fan of mixing extensions for different types in the same file

Reply:
OK, will split.